snakemake/__init__.py

```diff
@@ -1135,6 +1139,11 @@ def get_argument_parser(profile=None):
         help="Cleanup old shadow directories which have not been deleted due "
         "to failures or power loss.",
     )
+    group_utils.add_argument(
+        "--skip-cleanup-scripts",
```

Comment:
Thought about it. Let us call it `--skip-script-cleanup` instead. Otherwise we risk that people misunderstand this as an option that skips some kind of cleanup scripts :-).

Reply:
Done.